delta/app/src/test/scala/ch/epfl/bluebrain/nexus/delta/routes/ResourcesRoutesSpec.scala

```diff
@@ -183,13 +178,20 @@ class ResourcesRoutesSpec extends BaseRouteSpec with IOFromMap {
     "fail to create a resource that does not validate against a schema" in {
       Put(
         "/v1/resources/myorg/myproject/nxv:myschema/wrong",
-        payload.removeKeys(keywords.id).replaceKeyWithValue("number", "wrong").toEntity
+        payloadWithoutId.replaceKeyWithValue("number", "wrong").toEntity
```

Comment:
After looking at this for a while I can see now that if you set the number field to be a string, presumably that breaks validation. It's not very obvious though, because there is nothing to suggest that is what the schema checks. With this and other tests, it's possible to make it clearer

Reply:
I tried to make it a bit clearer, note that this was some existing test code, that I only refactored a bit